tests/nested.rs

```diff
@@ -53,7 +50,7 @@ fn nested_middleware() {
         app.middleware(test_middleware);
         app.at("/echo").get(echo_path);
         app.at("/:foo/bar").strip_prefix().get(echo_path);
-        route.strip_prefix().get(app.into_http_service());
```

Comment:
Should we remove the `strip_prefix` method too? Or are there any other uses left for it still?

Reply:
I left the method for `HttpService`s other than Tide. Users might want to nest some service and call `route.strip_prefix().all(my_http_service)`.

Reply:
Alright, fair enough!